File: src/main/java/org/noxet/noxetserver/util/Team.java
package org.noxet.noxetserver.util;

import org.bukkit.ChatColor;
import org.bukkit.Material;

public interface Team {
    String getTeamId();
    String getDisplayName();
    String getFormattedDisplayName();
    ChatColor getColor();
    int getMaxTeamMembers();
    Material getTeamIcon();
}

File: src/main/java/org/noxet/noxetserver/util/Team.scala
package org.noxet.noxetserver.util

import org.bukkit.{ChatColor, Material}

trait Team:
  def getTeamId: String

  def getDisplayName: String

  def getFormattedDisplayName: String

  def getColor: ChatColor

  def getMaxTeamMembers: Int

  def getTeamIcon: Material